tests/users/test_api_list_users.py

```diff
@@ -136,6 +137,18 @@ def test_list_users_api_with_search_with_special_characters_resource_auth(self):
         self.assertEqual(200, actual_response.status_code)
         self.assertEqual(expected_response, json.loads(actual_response.data))
 
+    def test_list_users_api_with_skills_resource_auth(self):
+        auth_header = get_test_request_header(self.admin_user.id)
+        expected_response = [marshal(self.second_user, public_user_api_model)]
+        actual_response = self.client.get(
+            f"/users?skills={self.second_user.skills}",
+            follow_redirects=True,
+            headers=auth_header,
+        )
+
+        self.assertEqual(200, actual_response.status_code)
+        self.assertEqual(expected_response, json.loads(actual_response.data))
+
```

Comment:
can you add another test, for testing if it searches for a skill, within a list of skills. Example is: skills = "kotlin, python, cooking" if the skills searched is python, it should return the user.

Reply:
Thanks. I will add the testcase tomorrow.